decidim-core/spec/system/session_timeout_spec.rb

```diff
@@ -29,15 +29,23 @@
       switch_to_window(open_new_window)
       visit decidim.root_path
       2.times.each do
-        expect(page).to have_content("If you continue being inactive", wait: 3)
+        expect(page).to have_content("If you continue being inactive", wait: 4)
         find("#continueSession").click
         travel 20.seconds
       end
 
       switch_to_window(win1)
-      expect(page).to have_content("If you continue being inactive", wait: 3)
+      expect(page).to have_content("If you continue being inactive", wait: 4)
       find("#continueSession").click
       expect(page).not_to have_content("You were inactive for too long")
     end
+
+    it "doesnt timeout user who wants to be remembered" do
+      current_user.remember_created_at = Time.current
```

Comment:
I'm not sure why this test actually works but shouldn't we update the database record here instead of setting the `remember_created_at` temporarily to RAM?